ansible/roles/local_storage/templates/node.yml.j2

```diff
@@ -0,0 +1,9 @@
+{% for node in item['nodes'] %}
+---
+apiVersion: v1
+kind: Node
+metadata:
+  name: {{node}}
+  labels:
+    {{item.label}}
```

Comment:
I think when we re-organized the playbooks such that labeling was a separate playbook, I might have forgotten to remove the [task](https://github.com/redhat-performance/webfuse/blob/main/ansible/roles/local_storage/tasks/main.yml#L4-L8) that maps to this file.  I don't see `local_storage_nodes` defined in the sample vars file so it is likely to have always just skipped that task with the missing file.  If we add the file/function back in I think we should at least have the var in the sample file, what do you think?

Reply:
yeah you are right, I think we should just stay with that separate playbook `label-node.yml` using `label_node_values` inputs. Then probably this tasks is obsolete and never been called anytime, I will remove the tasks and update the PR.